src/main/java/com/github/relativobr/supreme/generic/electric/EnergyGenerator.java

```diff
@@ -83,11 +86,22 @@ protected int[] getOutputSlots() {
   @Override
   public int getGeneratedOutput(Location l, Config data) {
 
-    int gen = this.type.generate(l.getWorld(), l.getBlock(), this.energy);
+    if(this.generate > 0 && (this.currentDelay < Supreme.getSupremeOptions().getDelayTimeValidGenerators())){
+
+      this.currentDelay++;
+
+    } else {
+
+      // check block
+      this.generate = this.type.generate(l.getWorld(), l.getBlock(), this.energy);
+      this.currentDelay = 0;
+
```

Comment:
Are the empty lines really necessary?

```suggestion
      this.currentDelay++;
    } else {
      // check block
      this.generate = this.type.generate(l.getWorld(), l.getBlock(), this.energy);
      this.currentDelay = 0;
```

Reply:
they are not